x-pack/plugins/uptime/server/rest_api/synthetics_service/get_service_locations.ts

```diff
@@ -14,5 +14,5 @@ export const getServiceLocationsRoute: UMRestApiRouteFactory = () => ({
   path: API_URLS.SERVICE_LOCATIONS,
   validate: {},
   handler: async ({ server }): Promise<any> =>
-    getServiceLocations({ manifestUrl: server.config.service.manifestUrl }),
+    getServiceLocations({ manifestUrl: server.config.unsafe.service.manifestUrl }),
```

Comment:
told you this type isn't working. Something weird is going on, i will investigate.